pkg/workceptor/stdio_utils.go

```diff
@@ -27,7 +27,16 @@ type stdoutWriter struct {
 
 // newStdoutWriter allocates a new stdoutWriter, which writes to both the stdout and status files
 func newStdoutWriter(unitdir string) (*stdoutWriter, error) {
-	writer, err := os.OpenFile(path.Join(unitdir, "stdout"), (os.O_APPEND|os.O_CREATE)+os.O_WRONLY+os.O_SYNC, 0600)
+	var osFlags int
+	stdoutPath := path.Join(unitdir, "stdout")
+	if _, err := os.Stat(stdoutPath); err == nil {
```

Comment:
my assumption here is that if the stdout exists at all (regardless of its content), we are safe to open without the create flag